### DISCUSSION TITLE
Website ChatGPT Error

### DISCUSSION THREAD

**Discussion by bluey656yt** [Zero6992/chatGPT-discord-bot#229] [Q&A] [answered]
The official 3.5 model works fine but when I switch to the unofficial model, I get this error in console:

![image](https://user-images.githubusercontent.com/120713465/223413144-67b47159-f50c-4a2e-86e3-ed89cf4f482a.png)

Any idea why?

**Comment by Zero6992** [comment, marked as answer]
`pip3 install revChatGPT==3.1.5` 

make sure your revChatGPT is 3.1.5

**Reply by bluey656yt** [reply]
That fixed it, thanks!
